recipes/r-sceasy/meta.yaml

```diff
@@ -24,10 +24,11 @@ requirements:
         - r-base
         - r-reticulate
         - r-seurat >=3.0.1
```

Comment:
Doesn't the current version require some 4.x Seurat version @nh3? or will it still work with Seurat 3.x?

Reply:
It should still work with 3.x, I think.